torch/csrc/cuda/CUDAPluggableAllocator.h

```diff
@@ -66,11 +38,11 @@ struct _AllocationMetadata {
   cudaStream_t stream{};
 };
 
-struct TORCH_CUDA_CPP_API CUDAPluggableAllocator
```

Comment:
Same here: please add the TORCH_CUDA_CPP_API so that cpp code can use these APIs.